OpenEdXMobile/src/main/java/org/edx/mobile/base/MainApplication.java

```diff
@@ -142,6 +144,16 @@ private void init() {
         registerReceiver(new NetworkConnectivityReceiver(), new IntentFilter(ConnectivityManager.CONNECTIVITY_ACTION));
         registerReceiver(new NetworkConnectivityReceiver(), new IntentFilter(WifiManager.WIFI_STATE_CHANGED_ACTION));
 
+        //Media status update receiver required for the SD Card storage feature
+        if (Build.VERSION.SDK_INT > Build.VERSION_CODES.KITKAT) {
```

Comment:
We should have declared in AndroidManifest because of technical reasons.
We want these callbacks even MainApplication instance is not in memory.